File: Java/bee 1005.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double A, B;
        A = sc.nextDouble() * 3.5;
        B = sc.nextDouble() * 7.5;
        double MEDIA = (A + B)/11;

        System.out.printf("MEDIA = %.5f\n", MEDIA);
    }
}

File: Kotlin/bee 1005.kt
import java.util.*
import java.io.BufferedReader
import java.io.InputStreamReader

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val a = br.readLine().toDouble() * 3.5
    val b = br.readLine().toDouble() * 7.5

    println(String.format("MEDIA = %.5f", (a+b)/11))
	
}
